test/jasmine/tests/select_test.js

```diff
@@ -98,11 +98,11 @@ function assertEventCounts(selecting, selected, deselect, msg) {
 
 // events for box or lasso select mouse moves then a doubleclick
 var NOEVENTS = [0, 0, 0];
-// deselect gives an extra plotly_selected event on the first click
-// event data is undefined
-var BOXEVENTS = [1, 2, 1];
+// deselect used to give an extra plotly_selected event on the first click
+// with undefined event data - but now that's gone, since `clickFn` handles this.
```

Comment:
Oh. Some users might currently rely on `plotly_selected` sending `undefined` as event data.

Reply:
put back the old behavior in https://github.com/plotly/plotly.js/pull/2241/commits/974cebf4e6f596d4fde8c84053aef43100b2ea10 and commented in #420 about removing it later.